app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -201,12 +204,13 @@ private fun MessageComposer(
                                 )
                             }
                             .background(color = colorsScheme().backgroundVariant)
+                            .padding(bottom = if (isConversationMember) 0.dp else dimensions().spacing16x)
                             .weight(1f)) {
                         content()
                     }
                     if (isUserBlocked) {
                         BlockedUserMessage()
-                    } else {
+                    } else if (isConversationMember) {
```

Comment:
Would this mean that if the other user is blocked, this condition will not be checked?

Reply:
Also `isUserBlocked` is only valid for 1:1 conversations or for group conversations as well?

Reply:
cc @borichellow

Reply:
`isUserBlocked ` only for 1:1 conversations